app/controllers/events_controller.rb

```diff
@@ -1,6 +1,7 @@
 class EventsController < ApplicationController
 
   def index
+    @events = Event.all.paginate(page: params[:page])
   end
```

Comment:
I think we can just scope by status: "upcoming" and date in the future. Admin is able to see all events, but user won't need to.